add Kotlin offline mode (#135)
amplitude/amplitude-dev-center: docs/data/sdks/android-kotlin/index.md

    context = applicationContext,
))
```

### Offline mode

> Starting from version 1.13.2, Amplitude Android Kotlin SDK supports offline mode out of box. The SDK checks network connectivity every time when an event is tracked. If it's connected to network, a flush will be schedule. If not, the event will be saved to storage. The SDK also listen to network change. It flushes all stored events when network is back.

Reviewer: ```suggestion
Starting from version 1.13.2, the Amplitude Android Kotlin SDK supports offline mode. The SDK checks network connectivity every time it tracks an event. If the device is connected to network, the SDK schedules a flush. If not, it saves the event to storage. The SDK also listens for changes in network connectivity and flushes all stored events when the device reconnects.
```

    context = applicationContext,
))
```

### Offline mode

Starting from version 1.13.0, the Amplitude Android Kotlin SDK supports offline mode. The SDK checks network connectivity every time it tracks an event. If the device is connected to network, the SDK schedules a flush. If not, it saves the event to storage. The SDK also listens for changes in network connectivity and flushes all stored events when the device reconnects.

> To enable this feature, you need to add `ACCESS_NETWORK_STATE` permission to `AndroidManifest.xml`. Otherwise, the SDK will just flush the event based on `flushIntervalMillis` and `flushQueueSize`.

Reviewer: ```suggestion
To enable this feature, add the `ACCESS_NETWORK_STATE` permission to `AndroidManifest.xml`. Otherwise, the SDK flushes the event based on `flushIntervalMillis` and `flushQueueSize`.
```

```xml
<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

You can also implement you own offline logic:

> 1. Disable the default offline logic by setting `config.offline` to `AndroidNetworkConnectivityCheckerPlugin.Disabled`

Reviewer: ```suggestion
1. Set `config.offline` to `AndroidNetworkConnectivityCheckerPlugin.Disabled` to disable the default offline logic.
```
